lib/livingdocs_connector.js

```diff
@@ -0,0 +1,68 @@
+const _ = require('lodash')
+const axios = require('axios')
+const {mapMetadata} = require('../mappers/image_mapper')
```

Comment:
I would separate the Livingdocs API and the creation of the document

Reply:
Once, we have the import in the SDK, the API part will be in the SDK.